### ISSUE TITLE
python proxy not work at all

### ISSUE THREAD

**Issue by A2116** [alexbers/mtprotoproxy#244]
python proxy does not work on any port and configuration
the official work fine on the same server
it prompts on service following line
Unable to connect to 2001:67c:04e8:f004::a 443
also if I change to python 3.6
os is centos 8.3

**Comment by mmdation** [comment]
Same here on ubuntu 18.04

**Comment by jjjan** [comment]
yes me too it's try to connect to ipv6 instead of ipv4. if you check code you find out the code first  connect to ipv6 first and if does not exist try v4 and the loop goes on.

**Comment by jjjan** [comment]
I changed the code enjoy https://github.com/jjjan/mtprotoproxy1 
halesho bebarin

**Comment by Mee1ad** [comment]
> I changed the code enjoy https://github.com/jjjan/mtprotoproxy1
> halesho bebarin

thanks bro
damet garm agha :)

**Comment by seriyps** [comment]
Another option is to add `PREFER_IPV6 = False` to config.py

**Comment by alexbers** [comment]
hi, the proxy tries to use ipv6 by default if it is available.

I added one more check if ipv6 is in good condition to use it: https://github.com/alexbers/mtprotoproxy/commit/6e8e8b63b26507310bbe550d5fbd6851f598f6e1 , hope it helps

**Comment by noorbala7418** [comment]
hi @alexbers !
Same here this problem on ubuntu 18.04 LTS.
I used Last version of yout code but didn't.

finally I added `PREFER_IPV6 = False` on top of `config.py` and it worked for me.

thanks
